b2share/modules/records/cli.py

```diff
@@ -54,6 +55,14 @@ def check_dois(update):
         check_record_doi(record, update)
 
 
+@b2records.command()
+@with_appcontext
+def update_expired_embargoes_now():
```

Comment:
The code uses explicitly `delay`, IMHO `update_expired_embargoes_now` is still a little confusing as we don't really update it "now".